h2o-genmodel/src/main/java/hex/genmodel/algos/drf/DrfMojoModel.java

```diff
@@ -58,4 +62,47 @@ public DrfMojoModel(String[] columns, String[][] domains, String responseColumn)
         return score0(row, 0.0, preds);
     }
 
+    @Override
+    public PredictContributions makeContributionsPredictor() {
```

Comment:
again - seems duplicated

Reply:
Yes, main diff is just how we handle binomial and regression in DRF vs. GBM.